web-client/public/js/grnsight.js

```diff
@@ -1,3 +1,6 @@
+import setupHandlers from "setuphandlers";
+import updateApp from "updateapp";
+import grnState from "grnstate";
```

Comment:
You import these but aren't calling. Make sure that the code all works with this in the code:
```javascript
setupHandlers();
updateApp(grnState);
```